components/layout/index.jade

```diff
@@ -4,9 +4,16 @@ html( lang="zh" )
     title Artsy
     meta( name="viewport" content="width=device-width" initial-scale="1" )
     meta( http-equiv="Content-Type" content="text/html; charset=UTF-8" )
+    link( type='text/css', rel='stylesheet', href='//fast.fonts.net/cssapi/f7f47a40-b25b-44ee-9f9c-cfdfc8bb2741.css' )
     link( rel='stylesheet' type='text/css' href='index.css' )
   body
     header.header
+      //- Rendered client-side
+
     .body-content-container.main-side-margin
+      //- Rendered client-side
+
     footer.footer
-    script( src='index.js' )
\ No newline at end of file
+      //- Rendered client-side
```

Comment:
I think there should be just one big template you re-render; I also think this can be compiled pre-rendered rather than having it be client-side from the outset